### ISSUE TITLE
T-7A mod does not work in DCS 2.8.x

### ISSUE THREAD

**Issue by plott1964** [ufo2321/T-7A-Red-Hawk-Community-Mod-For-DCS#1]
ufo2321 - The T-7A mod does not work in DCS 2.8.x. Just wanted you to be aware. Please note I'm willing to assist get this working.

**Comment by ufo2321** [comment]
> ufo2321 - The T-7A mod does not work in DCS 2.8.x. Just wanted you to be aware. Please note I'm willing to assist get this working.

Hi my first "fan"😅. Yes, sadly the mod stopped working in 2.8. I will probably have to start over with this mod. Although that will be in the future as right noe I'm working on the JAS-39C Gripen mod :) But I'll revive the project in the future, stay tuned and can I get your discord so I can message you when that happens?

**Comment by plott1964** [comment]
I just wanted to shoot you a message in support of all the hard work you do on your mods (much appreciated) and wanted to ask if you would be interested in my support/assistance with your mod development.

I have been in the Aerospace industry (McDonnell Douglas/Boeing) for over 28 years and having been supporting in training efforts as an illustrator and courseware programmer on multiple platforms including the F-15, F-18, T-45, V-22, P-8 and various others. I would love to assist you as you see fit with beta testing your mods or whatever you would need. Just let me know. I'm currently a beta tester for Spino and his modules.

I had weird trouble with the Gripen as well. When I had it loaded it would causes a super carrier to disappear and the aircraft on it to crash into the water, even though the Gripen wasn't even used in the mission. I know this sounds very weird. I can send the file if you want to help troubleshoot, because I really like the Gripen to. You should be able to find me on discord as infidel1964.

**Comment by ufo2321** [comment]
> I just wanted to shoot you a message in support of all the hard work you do on your mods (much appreciated) and wanted to ask if you would be interested in my support/assistance with your mod development.
> 
> I have been in the Aerospace industry (McDonnell Douglas/Boeing) for over 28 years and having been supporting in training efforts as an illustrator and courseware programmer on multiple platforms including the F-15, F-18, T-45, V-22, P-8 and various others. I would love to assist you as you see fit with beta testing your mods or whatever you would need. Just let me know. I'm currently a beta tester for Spino and his modules.
> 
> I had weird trouble with the Gripen as well. When I had it loaded it would causes a super carrier to disappear and the aircraft on it to crash into the water, even though the Gripen wasn't even used in the mission. I know this sounds very weird. I can send the file if you want to help troubleshoot, because I really like the Gripen to. You should be able to find me on discord as infidel1964.

Yes that would be awesome! I'll make sure to contact you when I get going on this mod!

The super carrier bug is a known bug, we'll fix it in the future. A temporary "fix" for that is to move the "JAS39" folder when you won't use the Gripen :)
